basic-features.md

```diff
@@ -1,80 +1,149 @@
+| [Views](/views.md)                                                                                       | Y            | Y            | Y            | Y            |
+| [Sequences](/sql-statements/sql-statement-create-sequence.md)                                            | Y            | Y            | Y            | Y            |
+| [Auto increment](/auto-increment.md)                                                                     | Y            | Y            | Y            | Y            |
+| [Auto random](/auto-random.md)                                                                           | Y            | Y            | Y            | Y            |
+| [DDL algorithm assertions](/sql-statements/sql-statement-alter-table.md)                                 | Y            | Y            | Y            | Y            |
+| Multi schema change: add column(s)                                                                       | Y            | Y            | Y            | N            |
+| [Change column type](/sql-statements/sql-statement-modify-column.md)                                     | Y            | Y            | N            | N            |
+| **Transactions**                                                                                         | **5.2**      | **5.1**      | **5.0**      | **4.0**      |
+| [Async commit](/system-variables.md#tidb_enable_async_commit-new-in-v50)                                 | Y            | Y            | Y            | N            |
+| [1PC](/system-variables.md#tidb_enable_1pc-new-in-v50)                                                   | Y            | Y            | Y            | N            |
+| [Large transactions (10GB)](/transaction-overview.md#transaction-size-limit)                             | Y            | Y            | Y            | Y            |
+| [Pessimistic transactions](/pessimistic-transaction.md)                                                  | Y            | Y            | Y            | Y            |
+| [Optimistic transactions](/optimistic-transaction.md)                                                    | Y            | Y            | Y            | Y            |
+| [Repeatable-read isolation (snapshot isolation)](/transaction-isolation-levels.md)                       | Y            | Y            | Y            | Y            |
+| [Read-committed isolation](/transaction-isolation-levels.md)                                             | Y            | Y            | Y            | Y            |
+| **Partitioning**                                                                                         | **5.2**      | **5.1**      | **5.0**      | **4.0**      |
+| [Range partitioning](/partitioned-table.md)                                                              | Y            | Y            | Y            | Y            |
```

Comment:
I think it's actually not GA, or maybe we need add some notes for partitioning if we want to mark it as GA? @bb7133

Reply:
IMHO we have fixed most of the bugs and it is fine to keep it 'GA' without a note here.